Index: mv_common.js
===================================================================
--- mv_common.js	(revision 38170)
+++ mv_common.js	(working copy)
@@ -57,6 +57,7 @@
 			base_offset=0;			
 		//js_log('set base offset: '+track_dur +'* .25 = '+ parseInt(base_offset) );
 	}
+	
 	js_log('BASE OFFSET: '+ base_offset);
 	//set the base offset / track_dur interface vars:
 	$j('#track_time_start_'+mvd_id).html( seconds2ntp(base_offset) );
@@ -139,6 +140,8 @@
 			do_video_time_update($j('#mv_start_hr_'+mvd_id).val(), $j('#mv_end_hr_'+mvd_id).val() );			
 		},
 		resize: function(e,ui) {	
+			base_offset = ntp2seconds( $j('#track_time_start_'+mvd_id).html());
+			//js_log("baseoffset: " + base_offset);
 			mv_slider_update_stats(mvd_id);
 		}		
 	});	
